cmd/builder/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:alpine
+
+WORKDIR /build
+
+# Copy default builder config and ocb binary into the container
+COPY builder-config.yaml /build/builder-config.yaml
+COPY --chmod=755 ocb /build/ocb
+
+ENTRYPOINT [ "./ocb"]
+# default arguments to use provided builder config file from documentation;
+# replace if using a different file location or file name
+CMD [ "--config", "builder-config.yaml" ]
```

Comment:
Let's also remove the `CMD` instruction and let the user specify the `--config` flag and other options themselves. This will make the behavior of the container the same as that of the binary itself.

The documentation will probably need an example on how to:
- pass the configuration to the builder,
- get the built sources and/or binary out of the container into the user's host machine.

Reply:
done, thanks. will also update the draft documentation I am working to raise that PR once this is accepted